.github/workflows/e2e-parallel-destroy.yml

```diff
@@ -51,11 +50,11 @@ jobs:
             tenant_name: 'nginx'
             deployment_order:
               [
-                'module.e2e_test.module.aws_vpc',
+                'module.e2e_test.module.vpc',
                 'module.e2e_test.module.eks_blueprints',
                 'module.e2e_test.module.eks_blueprints_kubernetes_addons',
-                'module.e2e_test.module.aws_load_balancer_controller',
-                'module.e2e_test.module.ingress_nginx',
+                'module.e2e_test.module.eks_blueprints_kubernetes_addons.module.aws_load_balancer_controller[0]',
```

Comment:
Does this exclude LBC from being provisioned as part of `eks_blueprints_kubernetes_addons` target apply?

Reply:
Thought I will defer to @Zvikan to make sure I am doing everything properly for the GitHub actions workflow